test/aggregate_test.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-require 'test_helper'
```

Comment:
This is an interesting change, was this required?

Reply:
Yes, for some reason the file wasn't properly reading the `test_helper` for both rails 4 and 5